docs/cloud-soar/introduction.md

```diff
@@ -0,0 +1,670 @@
+---
+id: introduction
+title: Introduction to Cloud SOAR
+sidebar_label: Introduction
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+This article provides an introduction to Cloud SOAR for analysts and administrators.
```

Comment:
A suggestion -- if the user isn't sure of their role (analyst vs. admin), they could find out [here](https://help.sumologic.com/docs/get-started/onboarding-checklists/).

Reply:
Great suggestion, @kimsauce!